src/sentry/event_manager.py

```diff
@@ -1488,41 +1526,64 @@ def _save_aggregate(
                     state=GroupHash.State.LOCKED_IN_MIGRATION
                 ).update(group=group)
 
-                is_new = True
-                is_regression = False
+                is_new = False if seer_matched_group else True
```

Comment:
```suggestion
                is_new = bool(seer_matched_group)
```
<img src="https://media3.giphy.com/media/a5viI92PAF89q/giphy.gif"/>

Reply:
It would have to be `is_new = not bool(seer_matched_group)`. Is that clearer, do you think?